app/components/wds-avatar/avatar-badge.js

```diff
@@ -2,6 +2,11 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { inject as service } from '@ember/service';
 
+const userRoleMap = {
```

Comment:
missing tests for this

Reply:
https://github.com/Wikia/design-system/blob/master/tests/integration/components/wds-avatar-test.js#L28